lib/team_factory.rb

```diff
@@ -0,0 +1,19 @@
+class TeamFactory
+
+  def initialize
+    @teams = []
+  end
+  
+  def create_teams
+    CSV.foreach('./data/teams.csv', headers: true, header_converters: :symbol) do |row|
+      @teams << Team.new(
```

Comment:
Nice, created these objects the same way Noah did. I did it differently, but it all works!